patches/minecraft/net/minecraft/world/WorldServer.java.patch

```diff
@@ -71,14 +80,16 @@
          }
  
          this.field_72984_F.func_76318_c("tickPending");
-@@ -214,6 +224,10 @@
+@@ -214,6 +222,12 @@
          this.field_175740_d.func_75528_a();
          this.field_72984_F.func_76318_c("portalForcer");
          this.field_85177_Q.func_85189_a(this.func_82737_E());
 +        for (Teleporter tele : customTeleporters)
 +        {
 +            tele.func_85189_a(func_82737_E());
 +        }
++        this.field_72984_F.func_76318_c("chunkSource");
```

Comment:
[MCP SRG Mapper Chrome Plugin](https://chrome.google.com/webstore/detail/mcp-srg-mapper/ijjbogjfmngomjdmnabifgckbonmcebh?hl=en) has been updated recently. It helps with reading these patches.